README.md

```diff
@@ -1,65 +1,63 @@
-# Prometheus Exporter Snap for Hardware Observer charm
-
-
-You can learn more about Hardware Observer on [Charmhub](https://charmhub.io/hardware-observer).
-## Getting Started
-
-Install the snap from snap store:
-
-```bash
-$ sudo snap install prometheus-hardware-exporter --classic
-```
-
-Start the exporter:
-
-```bash
-$ sudo snap start prometheus-hardware-exporter
+# Prometheus Hardware Exporter
+
+Prometheus Hardware Exporter exports Prometheus metrics from BMCs (Baseboard
+management controllers) using IPMI (Intelligent Platform Management Interface)
+and Redfish protocol. It also exports and various SAS (Serial Attached SCSI) and
+RAID (redundant array of inexpensive disks) controllers.
```

Comment:
Would be nice to add a line mentioning that it's meant to be used alongside hardware observer. The line in the next paragraph only mentions "you can learn more about hardware observer..."

Reply:
Done